stack.yaml

```diff
@@ -10,6 +10,7 @@ packages:
 extra-deps:
 - base58-bytestring-0.1.0
 - generic-lens-1.1.0.0
+- esqueleto-2.7.0
```

Comment:
I am not sure we want to use esqueleto to be honest ...

Reply:
indeed!!!